portal/config/config.py

```diff
@@ -84,6 +84,8 @@ class BaseConfig(object):
     FLUSH_CACHE_ON_SYNC = True
 
     CELERY_IMPORTS = ('portal.tasks',)
+    CELERY_BEAT_PING_INTERVAL_SECONDS = 60 * 5
```

Comment:
Any thoughts on what these values should be?
CELERY_BEAT_PING_INTERVAL_SECONDS - how often celery beat pings our APIs
CELERY_BEAT_MISSED_PINGS_BEFORE_FAIL - the number of times we can miss a ping without failing

Reply:
@pbugni I should have given more background. I expect the ```/healthcheck``` request to originate from our monitoring service too. These variables don't control how often ```/healthcheck`` is called. They control how often we monitor celery beat.

There might be a better way to do this. If so, please let me know. Here's how I did it: I created a celery beat task that periodically (i.e. ```CELERY_BEAT_PING_INTERVAL```) calls ```/celery_beat_ping```. If that API isn't called within in a certain amount of time (i.e. ```get_celery_beat_threshold()```) we will assume celery beat is not running.

Reply:
@drryanjames I'm with you - no other obvious mech to confirm the *beat* service is functional.  This seems like a good solution.